src/pages/HomePage.js

```diff
@@ -0,0 +1,89 @@
+import {
+  ImageList, ImageListItem,
+  ImageListItemBar, Card, CardMedia,
+} from '@mui/material';
+import React, { useEffect } from 'react';
+import ArrowCircleDownRoundedIcon from '@mui/icons-material/ArrowCircleDownRounded';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import fetchImages from '../api/data_fetch';
+import '../components/image/image_card.css';
+import NavgationBar from '../components/NavgationBar';
+import Loading from '../components/Loading';
+
+export default function HomePage() {
+  const imagesData = useSelector((state) => state.imagesReducer);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (imagesData.images.length === 0) {
+      dispatch(fetchImages());
+    }
+  }, []);
+
+  if (imagesData.images.length === 0) {
+    return (
+      <div>
+        <Loading />
+      </div>
+    );
+  }
+  return (
+    <div className="home">
+      <NavgationBar />
+      <div style={{ marginTop: '63px', borderRadius: '0' }}>
+        <div className="image-tags">
+          <h4>
+            {
+              imagesData.images[0].tags.toUpperCase()
+            }
+          </h4>
+          <h5>
+            {
+              `${imagesData.images[0].likes.toLocaleString()} likes`
+            }
+          </h5>
+        </div>
+        <Card component={Link} to={`./details?id=${imagesData.images[0].id}`}>
+          <CardMedia
+            component="img"
+            height="194"
+            image={imagesData.images[0].webformatURL}
+            alt={imagesData.images[0].tags}
+          />
+        </Card>
```

Comment:
- [ ] Your intro banner is not descriptive of the list on the page. In the original design, the intro presents the total number of views and **`EUROPE`** which is a continent, thereby describing the regional the countries (Czech, Italy, etc) belong to. Your banner merely contains an image that does not describe the page. Kindly fix that.

  Your Implementation | Behance Design
  :---:|:---:
  ![akumu-descriptive](https://user-images.githubusercontent.com/36542986/163191781-13f7b463-695e-454f-9cdf-b4f68e60e228.jpg) | ![akumu-descr-local](https://user-images.githubusercontent.com/36542986/163191790-021ccb63-9794-4845-853e-8ec082602f78.jpg)